### PR TITLE
Kathleen database tests for left nav tabs and + dropdown

### PR DESCRIPTION
**Pull request by cheshirekate8**

### CODE REVIEW

File: cypress/integration/utils/sharedTests/repoDatabaseNav.ts [hunk 1]
```diff
@@ -12,7 +12,47 @@ const notExist = newShouldArgs("not.exist");
...
+      `[data-cy=tab-queries]`,
+      beVisible,
+    ),
+    newExpectationWithClickFlows(
+      "the Schema tab should be active",
+      `[data-cy=active-tab-schemas]`,
+      beVisible,
+      [newClickFlow(`[data-cy=tab-tables]`, [])],
+    ),
+  ],
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/repoDatabaseNav.ts:55]: The flow of these is a little off. I would expect it to be more like
```
[
  expectation for active tables tab,
  click flow for views with expectations for active views tab/inactive tables tab,
  click flow for queries tab with expectations for active queries tab/inactive views tab,
  click flow for schemas tab with expectations for active schemas tab/inactive queries tab
]
```

**Review by tbantle22 — APPROVED** [review]
Great job! Just had one comment about test flow. 

Also just general feedback - if you add more specific titles for your PRs it helps the reviewer know exactly what changes to expect/makes it easier to spot if we need to go back into merged PRs to find something. So for this something like "Database tests for left nav tabs and + dropdown" would be helpful

File: cypress/integration/utils/sharedTests/repoDatabaseNav.ts (hunk 1 above)
Comment (cheshirekate8) [reply cypress/integration/utils/sharedTests/repoDatabaseNav.ts]: Changed! Thank you!

Comment (tbantle22) [reply cypress/integration/utils/sharedTests/repoDatabaseNav.ts]: Awesome thanks they look good! This is good to merge once tests pass

**Pull request merged**